store/src/test/java/com/dropbox/android/external/store4/impl/FlowStoreTest.kt

```diff
@@ -325,7 +325,12 @@ class FlowStoreTest {
     fun diskChangeWhileNetworkIsFlowing_simple() = testScope.runBlockingTest {
         val persister = InMemoryPersister<Int, String>().asFlowable()
         val pipeline = StoreBuilder.from(
-            Fetcher.ofFlow { flow {} },
```

Comment:
This test was wrong before and tested the empty case (which changed in this diff). I updated it to emit something from fetcher to preserve the original meaning of the test

Reply:
👍